packages/daemon/daemon-docs.md

```diff
@@ -0,0 +1,112 @@
+# `@endo/cli`
+
+### `endo start`
+
+- executes [stop](#start) function definined within the `@endo/daemon`.
+
+### `endo stop`
+
+- executes [stop](#stop) function definined within the `@endo/daemon`.
+
+### `endo clean`
+
+- executes the [`clean`](#clean) function definined within the `@endo/daemon`.
+
+### `endo restart`
+
+- shorthand method for executing `endo stop`
+
+# `@endo/daemon`
+
+### `start`
+
+- handles setup of the initial process for creating Endo applications, and communicating across
+  which Endo applications can be created within. communicate.
+- this includes establishing an ipc channel which Endo applications can use to establish a connection
+  `makeEndoClient` to create a new connection.
+- The powers accessible within this new context are:
+
+  - `crypto`
+  - `net`
+  - `fs`
+  - `path`
+  - `popen`
+  - `url`
+
+- takes one argument, `locator`. If nothing is passed in, this defaults to [`defaultLocator`](#defaultlocator).
+
+### `terminate`
+
+- creates a new endo client using [makeEndoClient](#makeendoclient) for the sole purpose of gaining access to the `terminate` method.
+- Uses `terminate` in order to sever any existing connections.
+- takes one argument, `locator`. If nothing is passed in, this defaults to [`defaultLocator`](#defaultlocator).
+
+```js
+export const terminate = async (locator = defaultLocator) => {
+  const { resolve: cancel, promise: cancelled } = makePromiseKit();
+  const { getBootstrap, closed } = await makeEndoClient(
+    'harbinger',
+    locator.sockPath,
+    cancelled,
+  );
+  const bootstrap = getBootstrap();
+  await E(bootstrap)
+    .terminate()
+    .catch(() => {});
+  // @ts-expect-error zero-argument promise resolve
+  cancel();
+  await closed.catch(() => {});
+};
+```
+
+### `clean`
+
+- recursively and forcibly removes any directories that have been created throughout the lifecycle of the previous Endo daemon.
+- takes one argument, `locator`. If nothing is passed in, this defaults to [`defaultLocator`](#defaultlocator).
```

Comment:
Resets the Endo daemon’s persistent state to “factory defaults”. Actually an open question whether this will destroy the user’s identity. It might in fact do so, but maybe we’d require some “yes I mean it” incantation.

The purpose of the locator is to afford some flexibility for testing.